src/main/java/io/debezium/connector/jdbc/type/connect/ConnectTimeType.java

```diff
@@ -58,15 +49,23 @@ public void bind(Query<?> query, int index, Schema schema, Object value) {
             query.setParameter(index, null);
         }
         else if (value instanceof Date) {
-            final ZonedDateTime zdt = toZonedDateTime((Date) value);
-            query.setParameter(index, zdt, StandardBasicTypes.ZONED_DATE_TIME_WITHOUT_TIMEZONE);
+            final LocalTime localTime = DateTimeUtils.toLocalTimeFromUtcDate((Date) value);
+            final LocalDateTime localDateTime = localTime.atDate(LocalDate.now());
+            if (getDialect().isConnectionTimeZoneSet() || getDialect().isJdbcTimeZoneSet()) {
```

Comment:
Should not there be a default method like `isTimezoneAvailable()` or make `getDatabaseTimeZone` to return `Optional`?

Reply:
That was my original intent, but the behavior in `ZonedTimeType` differs based on which of these values are set individually, so I didn't want to necessarily introduce any confusion by doing this.  wdyt?